### PR TITLE
created About Section

### PR DESCRIPTION
**Pull request by imendy**
Screenshot 2023-10-07 055440
Screenshot 2023-10-07 055456
Screenshot 2023-10-07 055519
Screenshot 2023-10-07 055548
Screenshot 2023-10-07 055912

Screenshot 2023-10-07 055626

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -1,10 +1,10 @@
-@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap');
```
Comment (OlaOluwalekan) [inline client/src/App.css:1]: Nice. We are using quicksand front and not Lato?

**Review by OlaOluwalekan — APPROVED** [review]
Nice work. Can proceed to merge

File: client/src/App.css (hunk 1 above)
Comment (imendy) [reply client/src/App.css:1]: uhmm, i just thought Quicksand font looked better

Comment (OlaOluwalekan) [reply client/src/App.css:1]: Okay. No problem

Comment (OlaOluwalekan) [reply client/src/App.css:1]: > uhmm, i just thought Quicksand font looked better

The design use Lato though. But that's not a big deal. We can easily change it if we need to revert

**Pull request merged**